db/config.cc

```diff
@@ -1312,6 +1312,7 @@ std::map<sstring, db::experimental_features_t::feature> db::experimental_feature
         {"broadcast-tables", feature::BROADCAST_TABLES},
         {"keyspace-storage-options", feature::KEYSPACE_STORAGE_OPTIONS},
         {"tablets", feature::TABLETS},
+        {"raft-service-levels", feature::RAFT_SERVICE_LEVELS},
```

Comment:
We should try to make it automatic and directly dependent on group0. Otherwise we have yet another thing to test.

Reply:
I want to use this feature simply to use raft-based service level until there is no migration process. Since it's an experimental feature I'll remove it completely with the migration process code.

Reply:
removed the feature